elements/pfe-styles/demo/index.html

```diff
@@ -132,5 +163,72 @@ <h4>Code</h4>
       </section>
     </article>
 
+    <article>
+      <header>
+        <h2>Color Palette</h2>
+      </header>
+      <section>
+        <section class="pfe-l-grid pfe-m-gutters pfe-m-all-4-col">
+          <button class="themes" on="light" name="light">Light theme</button>
+          <button class="themes" on="dark" name="dark">Dark theme</button>
+          <button class="themes" on="saturated" name="saturated">Saturated theme</button>
```

Comment:
^ these buttons don't seem to work right, even with the pfe-colors.css file
![image](https://user-images.githubusercontent.com/456863/68345404-408da680-00bf-11ea-9dd0-74322c578d57.png)

Reply:
I kind of forgot I was playing with this page and stopped working on it mid-branch :)